doc/api/concatenated_openapi_spec.yaml

```diff
@@ -230,6 +240,8 @@ components:
             type: object
             description: 'Represents a quadruple of experiment, code, data and hardware-configuration'
             properties:
+                run_id:
```

Comment:
```suggestion
                runId:
```
We have been using camel case for all properties so far, please check everywhere in your PR for compliance

Reply:
I see. I used the naming-convention of mlflow for their Run entity, since being a sub-set was a requirement. On second glance, is makes more sense to be consistent with our conventions